### ISSUE TITLE
remove bundler dependency

### ISSUE THREAD

**Issue by mpapis** [nanoc/nanoc#516]
as discussed in #515 it might be worth to consider removing `bundler` dependency, the new `rubygems` (`2.0+`) supports `Gemfiles`

**Comment by ddfreyne** [comment]
What value does removing Bundler add, over just keeping it the way it is?

**Comment by mpapis** [comment]
less dependencies should be equal less problems ... assuming `rubygems` does what it says it does, it works most of the cases for me and it worked for `nanoc` - so I stand for less dependencies - btw. people can still use `bundler`, it just would not be a dependency any more as you can perform the same task with `rubygems` (which is part of every new `ruby` installation - all of which we support, as `1.8.7` is no more supported)

**Comment by ddfreyne** [comment]
I don’t think we’ve had any problems with Bundler so far. The only annoyance was that the preinstalled Bundler on Travis CI was slightly outdated.

I’m in the camp of “if it works, don’t touch it”.

**Comment by mpapis** [comment]
are we waiting on feedback from other contributors? or it's just postponed to review some time late? or should I close it? ;)

**Comment by ddfreyne** [comment]
I’m in favour of keeping the setup as-is, and closing the issue.
